Gemfile

```diff
@@ -1,4 +1,7 @@
+# frozen_string_literal: true
+
 source "https://rubygems.org"
 
-# Specify your gem's dependencies in ikusei-linters.gemspec
-gemspec
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```

Comment:
Was ist denn das für ein Trick?

Reply:
Sicherstellung das die Quellen per https eingebunden werden: https://stackoverflow.com/questions/41454333/meaning-of-new-block-git-sourcegithub-in-gemfile